src/MetadataUtility/Models/Recording.cs

```diff
@@ -89,7 +89,7 @@ public string SourcePath
         /// <summary>
         /// Gets the sample rate of the recording.
         /// </summary>
-        public uint SampleRateHertz { get; init; }
+        public ulong SampleRateHertz { get; init; }
```

Comment:
This definitely does not need to be a ulong 0 to 18,446,744,073,709,551,615. I don't know about you but the highest sample rate i've ever seen was 512,000 - roughly 36028797018963 times smaller than a ulong max